foldamers/cg_model/cgmodel.py

```diff
@@ -174,21 +84,15 @@ class CGModel(object):
     def __init__(
```

Comment:
Do we need to update the _built_in_regions here to match the changes? Not really sure what this part is for.

Reply:
Good question on the _BUILT_IN_REGIONS_.  There IS actually a purpose here, I figured out, but I'm not sure this is used correctly, and may not be useful now.  It's only defined here, and not used in the rest of the system.  Looks like in Yank, one could define built-in regions (see http://getyank.org/latest/_modules/yank/yank.html), but we're not using yank anymore, and in any case those would just be atom selections, not general model properties. So I will modify the PR and remove this.